### PR TITLE
New package releases

### PR DESCRIPTION
**Pull request by jessebraham**
Please review the changes to the version numbers, so make sure I have not made any silly mistakes 😁 I think things should be in order, though.

Tomorrow morning I will merge this PR and publish all relevant packages.

### CODE REVIEW

**Comment by jessebraham** [comment]
Added the `skip-changelog` label because CI was complaining about `esp-lp-hal`.

**Comment by bugadani** [comment]
The headings in the migration guides currently say `to vNext`, we should fix those to the actual version numbers.

File: esp-hal-embassy/MIGRATING-0.3.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
-Migration Guide from 0.3.x to vNext
-====================================
+# Migration Guide from 0.3.x to v0.4.x
 
-Initialsation
--------------
+## Initialsation
```
Comment (MabezDev) [inline esp-hal-embassy/MIGRATING-0.3.md:3]:
```suggestion
## Initialization
```

Pains me to use that version :D, but we may as well be consistent with the rest of our docs

**Review by MabezDev — APPROVED** [review]
LGTM, thanks!

**Pull request merged**